maven-embedder/src/main/java/org/apache/maven/cli/CLIManager.java

```diff
@@ -167,7 +170,7 @@ public void displayHelp( PrintStream stdout )
 
         HelpFormatter formatter = new HelpFormatter();
 
-        formatter.printHelp( pw, HelpFormatter.DEFAULT_WIDTH, "mvn [options] [<goal(s)>] [<phase(s)>]", "\nOptions:",
+        formatter.printHelp( pw, 120, "mvn [options] [<goal(s)>] [<phase(s)>]", "\nOptions:",
```

Comment:
I think we should replace those `\n` with `System.lineSeparator()`.

Reply:
+1